modules/oe_translation_poetry/src/Form/PoetryCheckoutFormBase.php

```diff
@@ -184,6 +184,25 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     // The submit handler is submitRequest().
   }
 
+  /**
+   * Returns the title of the form page.
+   *
+   * @return string
+   *   The title for the form page.
+   */
+  public function getPageTitle(): string {
+    $title = 'Send request to DG Translation';
```

Comment:
This needs to be translatable I think. Make sure it's in t(). And not `t($var)`. You can use placeholders for the languages.